scene/gui/color_picker.cpp

```diff
@@ -399,7 +399,7 @@ void ColorPicker::_screen_input(const InputEvent &ev)
 		Image img =r->get_screen_capture();
 		if (!img.empty())
 			last_capture=img;
-			r->queue_screen_capture();
```

Comment:
This one looks like a bug too, I think it was @MarianoGnu who worked on the ColorPicker? Could you take a quick look?